virus/Hemileia_pugnator/Hemileia pugnator.md

```diff
@@ -0,0 +1,132 @@
+# Hemileia pugnator: Brewing trouble for the Causative Agent of Coffee Leaf Rust disease, Hemileia vastatrix.
+written by: Flora HILT
+
+### Hemileia pugnator
+As with all newly characterized mitoviruses, the initial segment of this mitovirus name corresponds to the infected host, Hemileia vastatrix. Given its potential role in inducing a hypovirulent state in its host and recognizing that Hemileia vastatrix poses a significant threat to our beloved coffee, the concluding portion of its Latin name symbolizes its potential involvement in this battle, signifying "fighter." Now, let's meet Hemileia pugnator!
+
+![Hemileia pugnator - a Coffee Lover](H_pugnator_cover.jpg.jpg)
```

Comment:
Check filepath for all images `virus/Hemileia_pugnator/H_pugnator_cover.jpg.jpg`